class/defaults.yml

```diff
@@ -28,7 +28,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.11.2
+        tag: add/extensions
```

Comment:
Will be adjusted before merge.

Reply:
LGTM but please request again when the tag is adjusted